ansible/ci-build-collector.yml

```diff
@@ -40,10 +40,33 @@
       # ensure this action is printed
       tags: [print_action]
 
+    - block:
+      - name: Add drivers
+        community.docker.docker_image:
+          name: "{{ collector_image }}-full"
+          build:
+            platform: "linux/{{ arch }}"
+            target: probe-layer-1
+            args:
+              collector_repo: quay.io/stackrox-io/collector
+              collector_version: "{{ collector_tag }}"
+              module_version: "{{ driver_version }}"
+              max_layer_size: 300
+              required_layers: 0
+            path: "{{ context_path }}"
+
+      - name: Retag image
+        community.docker.docker_image:
+          name: "{{ collector_image }}-full"
+          repository: "{{ collector_image }}"
+          source: local
+
+      when: arch == "ppc64le"
```

Comment:
This goes with one of my previous comments, I don't think we need to be this fancy for the power image to get its drivers copied in. The main reason for the splitting in layers was, IIRC, that dockerhub had a sort of hard limit when pulling big layers and was causing time outs on the massive collector images. When we moved to quay.io, we had collector images that were 1GB+ on a single layer and we had no complains from users. On top of that, the power image is A LOT smaller, so I'm pretty sure we can just drop the entire layering mechanism and go for a direct copy of the kernel-modules directory.

Reply:
Yeah agreed. I've consolidated all this like you suggested and it's waaaay cleaner now